script/_dhcpv6_pd_ref

```diff
@@ -53,7 +54,13 @@ create_dhcpcd_conf_pd()
 noipv6rs # disable router solicitation
 interface ${UPSTREAM_INTERFACE}
   iaid 1
-  ia_pd 2/::/64 -
+  # Request a /64 prefix and assign it to ${WPAN_INTERFACE} with SLA ID 0
+  # and prefix length /64.
+  # This specific assignment (SLA ID 0 and matching /64 length) prevents
+  # dhcpcd from installing a reject route for the delegated prefix, which
+  # would otherwise make the route unusable after the /64 prefix is used by
+  # OpenThread to configure its mesh network.
```

Comment:
As discussed offline, please mention that this config will cause one more GUA on wpan0.